WordPress/twentytwentyfour
# Replace some images with placeholders

**Description**
Closes https://github.com/WordPress/twentytwentyfour/issues/392

**Screenshots**


**Testing Instructions**
Place the updated patterns in the block editor and confirm that there are no issues. Save and view the front. The images should not show.
